packages/lwc-language-server/src/template/__tests__/linter.test.ts

```diff
@@ -2,7 +2,7 @@ import { TextDocument } from 'vscode-languageserver';
 import templateLinter from '../linter';
 
 it('returns a list of all the template compilation errors', () => {
-    const content = `<template><template if:true="invalidExpression">{Math.random()}</template><lighting-card></lighting-card></template>`;
+    const content = '<template><template if:true="invalidExpression"></template>{Math.random()}<lighting-card></lighting-card></template>';
```

Comment:
It looks like LWC bails out early now because the `if:true` part is invalid and doesn't check the contents of the `if:true` block. Moving the `{Math.random()}` outside of the `if:true` block fixes this.